cli/dump.go

```diff
@@ -9,17 +9,42 @@ import (
 	cli "github.com/urfave/cli/v2"
 )
 
+type DumpFormat = string
+
+const (
+	dumpXml        DumpFormat = "xml"
+	dumpPostgreSQL DumpFormat = "postgres"
+	dumpSQLite     DumpFormat = "sqlite"
+	dumpMySQL      DumpFormat = "mysql"
+	dumpSQLServer  DumpFormat = "sqlserver"
+)
+
 func dump(c *cli.Context) error {
 	inputFile := c.Args().Get(0)
 	outputFile := c.Args().Get(1)
+	format := c.String("format")
 
 	if len(inputFile) == 0 || len(outputFile) == 0 {
 		return errors.New("usage: odict dump [input file] [output file]")
 	}
 
 	t(c, func() {
 		dict := odict.ReadDictionaryFromPath(inputFile)
-		dumped := dict.Dump()
+
+		// All SQL formats and XML
+		var dumped string
+		if format == dumpPostgreSQL {
```

Comment:
Nit: could be a bit cleaner if you used Go's built-in [switch syntax](https://go.dev/tour/flowcontrol/9).